src/shared/libfido2-util.c

```diff
@@ -323,6 +331,21 @@ static int fido2_use_hmac_hash_specific_token(
                 else
                         r = sym_fido_dev_get_assert(d, a, NULL); /* try without pin but with up now */
         }
+        if (uv_required && has_uv) {
+                r = sym_fido_assert_set_uv(a, FIDO_OPT_TRUE);
```

Comment:
let's just invoke this beforehand with the value encoded in the JSON data. i.e. if we know exactly what is configured we don#t have to try multiple times here